server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/routes/HealthCheckRoutesTest.java

```diff
@@ -65,7 +66,7 @@ public void setUp() throws Exception {
         healthChecks = new HashSet<>();
         webAdminServer = WebAdminUtils.createWebAdminServer(
             new DefaultMetricFactory(),
-            new HealthCheckRoutes(healthChecks));
+            new HealthCheckRoutes(healthChecks, new JsonTransformer()));
```

Comment:
You shoul be adding tests for `GET /healthcheck/checks` in this file, showing the  content of the response

Reply:
Added tests.